docs/routing/codable-routing.html

```diff
@@ -241,10 +239,10 @@ <h2 class="heading-2"><span class="blue-text">Step 5:</span> Create a GET ONE Co
         <pre><code>{"id": 1,"title":"Harry Potter","price":10.00,"genre":"Fantasy"}</code></pre>
         <div class="underline"></div>
 
-        <h2 class="heading-2"><span class="blue-text">Step 6:</span> Making codableStore thread safe (Optional)</h2>
+        <h2 class="heading-2"><span class="blue-text">Step 6:</span> Adding thread safety (Optional)</h2>
         <p class="block-text">Kitura route handlers are asynchronous.
-            If threads from multiple routes access the same object at the same time they will crash.
-            To prevent these collisions, we will serialize access to the `codableStore`.</p>
+            If threads from different routes try to change the same object at the same time they will crash.
+            To prevent these collisions in our example, we can serialize access to the `codableStore`.</p>
```

Comment:
As you're currentlyediting this file please can you make the Raw Routing guide on the line below into a link to that guide.